packages/crc64-nvme-crt/src/CrtCrc64Nvme.ts

```diff
@@ -0,0 +1,22 @@
+import { checksums } from "@aws-sdk/crt-loader";
+import { Checksum } from "@aws-sdk/types";
```

Comment:
minor: is the canonical type now in `@smithy/types`?

Reply:
Fixed in [`50f607a` (#6736)](https://github.com/aws/aws-sdk-js-v3/pull/6736/commits/50f607a3c6cb43bdd44e8d094e6c3a68e9dccbcc)